server/src/main/resources/application.properties

```diff
@@ -1,6 +1,6 @@
 server.port=${TESTSIGMA_SERVER_PORT:9090}
 server.url=${TESTSIGMA_SERVER_URL:https://local.testsigmaos.com}
-server.version=v2.9.1
+server.version=v2.9.2
```

Comment:
@vikramvr20 can we have the version as 3.0.0 because its major release

Reply:
Addressed